### DISCUSSION TITLE
Adding dewpoint to forecast

### DISCUSSION THREAD

**Discussion by ThePeePs** [poblabs/weewx-belchertown#783] [Ideas]
I was just looking at the forecast json, and had an idea.
There is space in the forecast next to the percentage of precipitation / snow amount to the option to have either the forecasted dew point, or rel. humidity percentage?

**Comment by poblabs** [comment]
That data comes from Aeris. If it's not there than it might not be available? Does weewx provide this data? I can't remember. If so it could go in weewx_data.json?

**Comment by ThePeePs** [comment]
Just double checked [Aeris's API doc](https://www.aerisweather.com/support/docs/api/reference/endpoints/forecasts/), and they do have both dewpoint[C|F] and humidity listed as part of the response

**Reply by poblabs** [reply]
I guess it should be added to the JSON 🙂

**Reply by ThePeePs** [reply]
I'll wait for others to chime in to see if something others would want too.  Should be pretty straight forward with the exception of the 1 hour forecast, cause of the way padding is done for the icons.

**Comment by ThePeePs** [comment]
Started playing with getting this added, if anyone is interested, you can view it [here](https://weather.thepeeps.net/dev).

Still very much a WIP.  Any suggestions or criticisms are welcome.

**Reply by fvirgola80** [reply]
The idea is to be able to add the values
wind direction, pressure, millimeters or inc for rain and uv in 24h 3h and 1h with commit #795   in addition to those already added for dew poit and humidity
@ThePeePs

**Comment by ThePeePs** [comment]
Think this is getting closer to being a PR, was able to get the 1 hour forecast to also have the dewpoint/humidity in as well. (please see my previous post for the link to my dev site)

**Comment by ThePeePs** [comment]
Submitted PR #795 for this idea

**Reply by poblabs** [reply]
Just merged it!
